frontend/src/scenes/onboarding/sdks/sdksLogic.tsx

```diff
@@ -39,10 +41,25 @@ Products that will often be installed in multiple places, eg. web and mobile
 */
 export const multiInstallProducts = [ProductKey.PRODUCT_ANALYTICS, ProductKey.FEATURE_FLAGS]
 
+const getOrderedSDKs = (sdks: SDK[]): SDK[] => {
+    return [
+        ...sdks.filter((sdk) => sdk.key === 'html'),
+        ...sdks.filter((sdk) => sdk.key === 'javascript-web'),
```

Comment:
No need to use `filter`, we can just find the original ones
```suggestion
        sdks.find((sdk) => sdk.key === 'html'),
        sdks.find((sdk) => sdk.key === 'javascript-web'),
```

You might need to add `.filter(Boolean)` at the end of this to make TS happy